src/components/SearchBox/optionsPopper.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from '@emotion/styled';
+import { Paper, PaperProps, Popper, PopperProps } from '@mui/material';
+
+export const OptionsPaper = styled(Paper)<PaperProps>`
+  background-color: ${({ theme }) => theme.palette.background.searchInput};
+  -webkit-backdrop-filter: blur(35px);
+  backdrop-filter: blur(35px);
```

Comment:
The custom component looks great on empty screen. Though i think, it shouldn't be transparent, when there is a sidebar open. Everything kinda melts together. Is it fixable?


<img width="401" alt="image" src="https://github.com/user-attachments/assets/4543d9ab-7e01-4687-ac58-98df662f0498" />